web-docs/architecture.md

```diff
@@ -0,0 +1,46 @@
+# Architecture
+
+flagd architectures fall into two broad categories: those where the evaluation engine is deployed in a standalone process to which the client application connects ([RPC](#rpc-evaluation)), and those where the evaluation engine is embedded into the client application ([in-process](#in-process-evaluation)).
+
+## RPC vs In-Process Evaluation 
+
+### RPC evaluation
+
+In RPC-based deployments one or more flagd instances deployed and exposed to client applications in your infrastructure.
+flagd RPC providers use HTTP or gRPC to request flag evaluations from flagd.
```

Comment:
Have we explained what a provider is by this point of the docs? Either way, we may want to create a glossary section with terms like provider.

https://squidfunk.github.io/mkdocs-material/reference/tooltips/?h=glo#adding-a-glossary-includesabbreviationsmd

Reply:
I think we should use the glossary in openfeature.dev for OpenFeature terms, but I would be OK with adding a glossary for flagd specific stuff.